doc/conf.py

```diff
@@ -61,8 +60,11 @@
 plot_html_show_source_link = False
 
 sphinx_gallery_conf = {
-    'examples_dirs': ['../examples', '../tutorials'],
-    'gallery_dirs': ['auto_examples']
+    'examples_dirs': ['../examples'],
+    'gallery_dirs': ['auto_examples'],
+    'doc_module': 'pyriemann-qiskit',
```

Comment:
I am not sure here. Should it be `pyriemann-qiskit` or `pyriemann_qiskit`?

Reply:
Well spotted!